tests/go.mod

```
@@ -33,6 +33,7 @@ require (
 	go.etcd.io/etcd/client/pkg/v3 v3.6.0-alpha.0
 	go.etcd.io/etcd/client/v2 v2.306.0-alpha.0
 	go.etcd.io/etcd/client/v3 v3.6.0-alpha.0
+	go.etcd.io/etcd/etcdctl/v3 v3.5.4
```

```suggestion
	go.etcd.io/etcd/etcdctl/v3 v3.6.0-alpha.0
```
This looks like it was not correctly imported.

---

so you are okay with importing `go.etcd.io/etcd/etcdctl/v3 ` in tests, i thought it would be an issue.
Can you help me in fixing this test case
```
FAIL: inconsistent versions for depencency: go.etcd.io/etcd/etcdctl/v3
  - github.com/bgentry/speakeasy@v0.1.0 from: go.etcd.io/etcd/etcdctl/v3
  - github.com/dustin/go-humanize@v1.0.0 from: go.etcd.io/etcd/etcdctl/v3
  - github.com/olekukonko/tablewriter@v0.0.5 from: go.etcd.io/etcd/etcdctl/v3
  - github.com/spf13/cobra@v1.4.0 from: go.etcd.io/etcd/etcdctl/v3
  - github.com/spf13/pflag@v1.0.5 from: go.etcd.io/etcd/etcdctl/v3
  - go.etcd.io/etcd/api/v3@v3.6.0-alpha.0 from: go.etcd.io/etcd/etcdctl/v3
  - go.etcd.io/etcd/client/pkg/v3@v3.6.0-alpha.0 from: go.etcd.io/etcd/etcdctl/v3
  - go.etcd.io/etcd/client/v3@v3.6.0-alpha.0 from: go.etcd.io/etcd/etcdctl/v3
  - go.etcd.io/etcd/etcdctl/v3@v3.5.4 from: go.etcd.io/etcd/tests/v3
  - go.etcd.io/etcd/etcdctl/v3@v3.6.0-alpha.0 from: go.etcd.io/etcd/v3
  - go.etcd.io/etcd/pkg/v3@v3.6.0-alpha.0 from: go.etcd.io/etcd/etcdctl/v3
  - golang.org/x/time@v0.0.0-20210220033141-f8bda1e9f3ba from: go.etcd.io/etcd/etcdctl/v3
  - google.golang.org/grpc@v1.41.0 from: go.etcd.io/etcd/etcdctl/v3
  - gopkg.in/cheggaaa/pb.v1@v1.0.28 from: go.etcd.io/etcd/etcdctl/v3
  - go.uber.org/zap@v1.17.0 from: go.etcd.io/etcd/etcdctl/v3
FAIL: inconsistent dependencies
FAIL: 'dep' failed at Sat Apr 30 12:59:18 PM IST 2022
```
https://github.com/etcd-io/etcd/pull/14000#issuecomment-1113943083

---

I haven't check if those functions are really needed, however I don't see a problem with importing `etcdctl` from `tests`. If you check https://github.com/etcd-io/etcd/blob/main/tests/go.mod#L10 it already has proper replace rule. 

I have a commit that successfully adds the dependency. https://github.com/serathius/etcd/commit/f44fd565f41a5c5a8f3bed64846ce4a500c6e2b6
What I did was add a `go.etcd.io/etcd/etcdctl/v3 v3.6.0-alpha.0` to tests/go.mod (please note version). And run `go mod tidy` in both `tests` directory and root.

---

updated `go.mod` as suggested